lib/faker/datetime.ex

```diff
@@ -15,30 +15,37 @@ if Version.match?(System.version(), ">= 1.3.0") do
     """
     @spec forward(integer) :: DateTime.t
     def forward(days) do
-      unix_from = DateTime.utc_now() |> DateTime.to_unix(:microseconds)
+      sign = if days < 0, do: -1, else: 1
+
+      today = DateTime.utc_now() |> to_timestamp
+      from = today + sign * @microseconds_per_day # add or subtract extra day to avoid returning today
+      to = from + @microseconds_per_day * days
 
-      unix_between(unix_from, days)
+      unix_between(from, to)
     end
 
     @doc """
-    Returns a random date between two dates, start date and end date not
-    included
+    Returns a random date between two dates, today included
```

Comment:
Nitpick: I would suggest to change this comment to inform the user it does also include a random time.

Something like `Returns a random DateTime's between two DateTime's` or `Returns a random date & time between two timestamps`